deployment/aides.sh

```diff
@@ -0,0 +1,124 @@
+#!/bin/bash
+
+#Variables du script
+ContainerName=AidesContainerLocal
+
+echo "Création du Container"
+echo ""
+lxc launch images:debian/stretch $ContainerName -c security.privileged=true
+echo ""
+
+echo "Pause de 15 secondes pour laisser le container démarrer"
+echo ""
+for i in {0..15}; do echo -ne "$i"'\r'; sleep 1; done; echo
+echo ""
+
+echo "Apt-get Update \ upgrade"
+echo ""
+lxc exec $ContainerName -- sh -c "echo "y\n" | apt-get update"
+lxc exec $ContainerName -- sh -c "echo "y\n" | apt-get upgrade"
+echo ""
+
+echo "Modification du passwd root (aides)"
+echo ""
+lxc exec $ContainerName -- sh -c "echo \"aides\naides\" | passwd"
+echo ""
+
+echo "Installation de ssh"
+echo ""
+lxc exec $ContainerName  -- sh -c "echo "y\n" |apt-get install ssh"
+echo ""
+
+echo "Installation de python"
+echo ""
+lxc exec $ContainerName  -- sh -c "echo "y\n" |apt-get install python"
+echo ""
+
+
+echo "Modification de sshd_config"
+echo ""
+lxc exec $ContainerName -- sh -c "sed -i \"s/#PermitRootLogin prohibit-password/PermitRootLogin yes/g\" /etc/ssh/sshd_config"
+echo ""
+
+echo "Redémarrage du service ssh"
+echo ""
+lxc exec $ContainerName -- sh -c "/etc/init.d/ssh restart"
+echo ""
+
+echo "Génération de la clé rsa, si elle n'existe pas"
```

Comment:
Attention, [toutes les clés ssh ne se trouvent pas forcément dans un fichier `id_rsa`](https://help.github.com/articles/checking-for-existing-ssh-keys/). Je suis mal à l'aise avec l'idée de gérer la création d'une clé ssh — qui est une fonctionnalité de sécurité importante — dans un script de création de vm, ça me semble hors sujet.

Vu que ce script s'adresse à des développeur·es, on peut partir du principe qu'ils disposent déjà d'une clé ssh à mon avis.

On peut aussi faire le test en début de script, et ne pas poursuivre si on ne trouve pas de clé publique, auquel cas on affiche un avertissement, mais sans créer la clé à la place de l'utilisateur.